common/network/components/BUILD

```diff
@@ -0,0 +1,38 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "broadcaster",
+    hdrs = ["broadcaster.h"],
+    deps = [
+        "//common/network/queues:concurrent_queue_trit_array",
```

Comment:
Would it look more concise to strip `/common/network/queues:concurrent_queue_trit_array` into `/common/network/queues:conqueue_trit_array` as [C++ Concurrent Queues](http://open-std.org/JTC1/SC22/WG21/docs/papers/2016/p0260r0.html) specifies the abbreviation `conqueues`?

Reply:
Why not, but it's only a matter of bazel as the actual type is typedefed down to something shorter